docs/how_to_guides/distributing_oci_layouts.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Distributing OCI Layouts
+sidebar_position: 6
+---
+
+# Distributing OCI Layouts
+
+### Phase 1: Creating an OCI Image 
```

Comment:
We need a consistent style for the sequential number. I seldom see articles use `phase` in their sub-headings.

Reply:
@FeynmanZhou I have just kept them as headings now. Kindly review.

Reply:
@deepeshaburse  if there are more fixes to title, please follow up as a updated PR.